docs/features/importing_n_files.md

```diff
@@ -0,0 +1,26 @@
+# Importing .n files
+
+This will be `import.n`
+```js
+// the pub modifier allows variables to be public
+
+let pub test = "Hello"
+let test2 = "Goodbye" // Cannot be accessed when imported
+
+print("test")
+```
+
+This will be `run.n`
+```js
+// imp takes in a string or a name (depricated) and returns a record after running though the file
+
+let importedthings = imp "./import.n" // This goes though the file so it will print out test
+
+print(importedthings.test) // Prints out hello
+//print(importedthigns.test2) will cause an error because it is not public
+```
+
+## Notes:
+- `imp` returns a record and so its type is as such.
```

Comment:
Technically it's not a record, but it's record-like. You can't assign `imp "./import.n"` to a variable of type `{ test: str }`

Reply:
Ok, added.